synapse/axon.py

```diff
@@ -569,12 +709,16 @@ def _findAxonClones(self):
 
                 self._initAxonClone(axfo[0])
 
+                waiter = self.waiter(1, 'syn:axon:clone:ready')
+                waiter.wait(60)
+
             except Exception as e:
-                logger.exception('findAxonClones')
+                logger.exception('Unhandled exception during _findAxonClones')
 
     def _findAxonClone(self):
```

Comment:
This function gives preference to host's with the lowest # of axons.  We still have a issue where, in a given set of axon hosts in a bus during startup; multiple hosts may slam the first host with a low count at first.